docs/developers/omnichain/tutorials/pyth.md

```diff
@@ -0,0 +1,40 @@
+# Pyth Price Oracle
+
+[Pyth Network](https://docs.pyth.network/) is a price oracle that provides a
+high-frequency, low-latency price feed for a wide range of assets. It is
+designed to be used by high-frequency trading systems and other applications
+that require a high degree of accuracy and reliability.
+
+Pyth is deployed on ZetaChain and provides price feeds for a wide range of
+assets, including cryptocurrencies, fiat currencies, and equity.
+
+Pyth contract addresses:
+
+| Network      | Address                                                                                                                                    |
+| :----------- | :----------------------------------------------------------------------------------------------------------------------------------------- |
+| Mainnet Beta | [0x2880aB155794e7179c9eE2e38200202908C17B43](https://zetachain.blockscout.com/address/0x2880aB155794e7179c9eE2e38200202908C17B43)          |
+| Testnet      | [0x0708325268dF9F66270F1401206434524814508b](https://zetachain-athens-3.blockscout.com/address/0x0708325268dF9F66270F1401206434524814508b) |
+
+Source: https://docs.pyth.network/price-feeds/contract-addresses/evm
+
+Pyth Network is a permissionless system, meaning that anyone can submit prices
+to the oracle. The system uses a decentralized consensus mechanism to ensure
+that the prices are accurate and reliable.
+
+ZetaChain runs a Pyth scheduler that submits prices to the Pyth oracle. The
```

Comment:
could we change it to: `The ZetaChain contributor team runs...`?

Reply:
f1c7f74